zpush/kweb.cfg

```diff
@@ -0,0 +1,23 @@
+:9080 {
+	log stdout
+	errors stderr
+
+	# healthcheck
+	status 200 /status
+
+	rewrite /Microsoft-Server-ActiveSync /Microsoft-Server-ActiveSync/index.php
+	fastcgi2 /Microsoft-Server-ActiveSync /run/php/php7.0-fpm.sock php {
+		without /Microsoft-Server-ActiveSync/
+		root /usr/share/z-push/
+		read_timeout 3605s
```

Comment:
The exact value should be exposed as configuration, probably also could be auto computed from the corresponding z-push configuration value at startup. (just a little higher than the max z-push ping time).